File: snippets/java/app/src/main/java/finetuning/ListTrainingStepMetrics.java
package finetuning;

import com.cohere.api.Cohere;
import com.cohere.api.resources.finetuning.finetuning.types.ListTrainingStepMetricsResponse;


public class ListTrainingStepMetrics {
    public static void main(String[] args) {
        Cohere cohere = Cohere.builder().token("<<apiKey>>").clientName("snippet").build();

        ListTrainingStepMetricsResponse response = cohere.finetuning().listTrainingStepMetrics("test-id");

        System.out.println(response);
    }
}

File: snippets/node/finetuning/list-training-step-metrics.js
import { CohereClient } from 'cohere-ai';

const cohere = new CohereClient({ token: process.env.COHERE_API_KEY! });

(async () => {
  const trainingStepMetrics = await cohere.finetuning.listTrainingStepMetrics(
    'test-finetuned-model-id'
  );

  console.log(trainingStepMetrics);
})();
